### PR TITLE
kubevirt: Use getGuestAgentFieldNotAvailMsg for users list

### PR DESCRIPTION
**Pull request by yaacov**
Align Guest Agent Field Not Avail Message in users list to use the new `getGuestAgentFieldNotAvailMsg` helper method.

### CODE REVIEW

**Comment by yaacov** [comment]
@pcbailey please review

File: frontend/packages/kubevirt-plugin/src/components/vms/vm-users.tsx [hunk 1]
```diff
@@ -69,12 +66,13 @@ export const VMUsersList: React.FC<VMUsersListProps> = ({ vmi, vmStatusBundle, d
...
-    return <div className="text-center">{VIRTUAL_MACHINE_IS_NOT_RUNNING}</div>;
-  }
+  const guestAgentFieldNotAvailMsg = getGuestAgentFieldNotAvailMsg(
+    isGuestAgentInstalled(vmi),
+    vmStatusBundle.status,
+  );
 
-  if (!isGuestAgentInstalled(vmi)) {
-    return <StatusItem Icon={BlueInfoCircleIcon} message={NO_GUEST_AGENT_MESSAGE} />;
+  if (!guestAgentInfoRaw && guestAgentFieldNotAvailMsg) {
```
Comment (irosenzw) [inline frontend/packages/kubevirt-plugin/src/components/vms/vm-users.tsx:74]: `guestAgentFieldNotAvailMsg` is redundant - it will always have a value no ?

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/vms/vm-users.tsx:74]: Right :exclamation: fixing

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/vms/vm-users.tsx]: Thanks :+1: fixed

**Pull request merged**